opentelemetry-sdk/tests/metrics/test_metrics.py

```diff
@@ -38,6 +38,26 @@ def test_resource_empty(self):
         # pylint: disable=protected-access
         self.assertIs(meter.resource, resources._EMPTY_RESOURCE)
 
+    def test_start_pipeline(self):
+        exporter = mock.Mock()
+        meter_provider = metrics.MeterProvider()
+        meter = meter_provider.get_meter(__name__)
+        # pylint: disable=protected-access
+        meter_provider.start_pipeline(meter, exporter, 6)
+        self.assertEqual(len(meter_provider._exporters), 1)
+        self.assertEqual(len(meter_provider._controllers), 1)
```

Comment:
Should it also call `meter_provider.shutdown()` to free that background thread it creates? Or patch out the `PushController` maybe.